app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -390,6 +420,13 @@ function AcuantCapture(
    */
   function startCaptureOrTriggerUpload(event) {
     if (event.target === inputRef.current) {
+      if (forceNativeCamera) {
```

Comment:
If I recall correctly, the layout for the field is different for the upload-only experience (eliminating the dual option of "Take a photo" _or_ "Upload from your device"). Should we trigger that UI when `forceNativeCamera` is in effect?

Reply:
After some discussion (and playing around), the team has decided to keep both of the options. So for now we don't need to make any other changes to the UI.